src/Operators/finitedifference.jl

```diff
@@ -1585,7 +1586,7 @@ end
         getidx(velocity, loc, idx, hidx),
         Geometry.LocalGeometry(space, idx, hidx),
     )
-    return Geometry.Contravariant3Vector(vᶠ)
+    return Geometry.Contravariant3Vector(zero(eltype(vᶠ)))
```

Comment:
@simonbyrne here I used `eltype` of `vᶠ`, which is a Contravariant3 and seems to be working fine. @Zhengyu-Huang in #887 used two `eltype`s. I think either approach works fine. Can you please confirm this?

Reply:
If this can work, then we can use only one `eltype`